### DISCUSSION TITLE
add websearch service Yandex

### DISCUSSION THREAD

**Discussion by korenevskiy** [Wox-launcher/Wox#3942] [General]
Please add websearch service "[Yandex](www.ya.ru)" in v2
`https://ya.ru/search/?text={q}`

**Comment by qianlifeng** [comment]
<img width="781" alt="image" src="https://github.com/Wox-launcher/Wox/assets/1725475/187afea0-34e5-4210-89f6-3d8543982c4f">

you can add custom websearch url in v1, do you mean this feature?

**Comment by korenevskiy** [comment]
> <img alt="image" width="781" src="https://user-images.githubusercontent.com/1725475/279399850-187afea0-34e5-4210-89f6-3d8543982c4f.png">
> you can add custom websearch url in v1, do you mean this feature?

YES, But to make it available in the software settings initially.
`https://ya.ru/search/?text={q}`
